File: 3226-minimum-number-game/minimum-number-game.py
class Solution:
    def numberGame(self, nums: List[int]) -> List[int]:
        nums.sort()
        for i in range(len(nums)-1):
            if i%2==0:
                temp = nums[i]
                nums[i]=nums[i+1]
                nums[i+1]=temp
        return nums
        

File: 3226-minimum-number-game/minimum-number-game.java
class Solution {
    public int[] numberGame(int[] nums) {
        Arrays.sort(nums);
        int t=0;
        for (int i = 0; i< nums.length;i+=2){
            t=nums[i];
            nums[i]=nums[i+1];
            nums[i+1]=t;
        }
        return nums;
        
    }
}